docs/orm-integration.md

```diff
@@ -272,6 +272,7 @@ In the generic buyer skill config (`my_buyer_aea/vendor/fetchai/skills/generic_b
 |ledgers: ['fetchai']               |ledgers: ['ethereum']             |
 |----------------------------------------------------------------------|
 ```
+
```

Comment:
I can change the format to the html table, but it would take.a bit of time. 
So wanted to double check, would that be what you want?
I don't know if that would solve the selection issue, will have to see.

Reply:
Don't change the table atm. We will deal with this another time.

Reply:
ok